ui/src/components/ReactJson.jsx

```diff
@@ -15,6 +15,9 @@ const useStyles = makeStyles({
 export default function ReactJson({ title, className, style, ...props }) {
   const [clipboardMode, setClipboardMode] = useState(false);
   const classes = useStyles();
+  
+  let modifiedProps = props && {src:{...props.src, tasks:props.src.tasks.filter(t => !["__start", "__final"].includes(t.taskReferenceName))}};
```

Comment:
nit: there's a lot going on here and it might be clearer broken out into a helper function or a few lines.

Reply:
thank you, created a separate method to take care of task filtering